content/blog/stackstorm-simple-elegant-automation-for-everyone.md

```diff
@@ -0,0 +1,33 @@
+---
+title: StackStorm, simple, elegant automation for everyone!
+date: 2021-11-18T15:16:44.594Z
+author: Rick Kauffman
+authorimage: /img/me-art-2.jpg
+---
```

Comment:
Please add tags: 
StackStorm
opensource
automation

Reply:
Not really sure how to add tags

Reply:
added tags